wdl/model/draft2/src/main/scala/wdl/draft2/model/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -325,7 +331,20 @@ object WdlStandardLibraryFunctions {
 
         // Inner function: Get the file size, allowing for unpacking of optionals
         def optionalSafeFileSize(value: WomValue): Try[Long] = value match {
-          case f if f.isInstanceOf[WomSingleFile] || WomSingleFileType.isCoerceableFrom(f.womType) => Try(Await.result(ioFunctionSet.size(f.valueString), Duration.Inf))
+          case f if f.isInstanceOf[WomSingleFile] || WomSingleFileType.isCoerceableFrom(f.womType) => {
+            val uuid = UUID.randomUUID().toString
+            val creation = OffsetDateTime.now
+            logger.info(f"($uuid) (0.000 seconds) optionalSafeFileSize $value start ($getClass) (thread '${Thread.currentThread().getName}')")
```

Comment:
is `value` going to give us something as useful as `f.valueString`?

Reply:
switched